global.R

```diff
@@ -12,9 +12,20 @@ source("functions.R")
 
 # database
 # loaded once per server rather than every client
-# proteomics <- dbConnect(RSQLite::SQLite(), "Proteomics.db")
 database <- dbConnect(RSQLite::SQLite(), "master_shiny_app.db")
 print("Connected to database")
 
 query <- paste0("SELECT * FROM master_database;")
 dataset <- dbGetQuery(database,query)
+
+# Generate all available plots
+available_plots <- new.env(hash=TRUE)
+available_plots[["tcga_aml"]] <- c("Multiplot","FAB", "Cyto_risk", "Fusion","Mutations")
+available_plots[["beat_aml"]] <- c("Multiplot","Fusion","Mutations")
+
```

Comment:
I certainly like this better than the config files!  I think it'd be possible to do this programmatically in the future, which would avoid the need for maintaining this list.  (run a db query, then ask "does tcga_aml have expression_data? If so, plots X and Y. Does it have fusion information, then plot Z", etc) in a generic way.   

That doesn't preclude this from being merged - Incremental improvements are good, and that's something we can add to the todo list in the future!